src/libs/Navigation/AppNavigator/AuthScreens.tsx

```diff
@@ -254,6 +258,14 @@ function AuthScreens({session, lastOpenedPublicRoomID, initialLastUpdateIDApplie
         return initialReport?.reportID ?? '';
     });
 
+    useEffect(() => {
+        RNNavBarManager.setButtonStyle(theme.navigationBarButtonsStyle);
```

Comment:
@kirillzyusko 

Should this only be for native platforms? Web crashes.

![Screenshot 2024-11-25 at 6 25 24 PM](https://github.com/user-attachments/assets/edab1a10-4d14-42d7-b674-d3db99260233)

![Screenshot 2024-11-25 at 6 25 36 PM](https://github.com/user-attachments/assets/6979052b-3153-4102-bcb2-a4019417a1f9)